packages/cardano-services-client/package.json

```
@@ -53,7 +53,7 @@
     "test:debug": "DEBUG=true yarn test"
   },
   "devDependencies": {
-    "@cardano-sdk/util-dev": "^0.8.0",
+    "@cardano-sdk/util-dev": "workspace:*",
```

Tilde range matches our pre-1.0 semver strategy
```suggestion
    "@cardano-sdk/util-dev": "workspace:~",
```